pkg/reconciler/taskrun/resources/taskref_test.go

```diff
@@ -532,11 +534,15 @@ func TestGetTaskFunc_RemoteResolution(t *testing.T) {
 		t.Fatalf("failed to get task fn: %s", err.Error())
 	}
 
-	resolvedTask, err := fn(ctx, taskRef.Name)
+	resolvedTask, annotation, err := fn(ctx, taskRef.Name)
 	if err != nil {
 		t.Fatalf("failed to call pipelinefn: %s", err.Error())
 	}
 
+	if !reflect.DeepEqual(expectedAnnotation, annotation) {
```

Comment:
We tend to prefer `if d := cmp.Diff(a, b); d != "" { t.Errorf("annotations did not match: %s", diff.PrintWantGot(d)) }`